processing/src/main/java/org/apache/druid/query/ordering/StringComparators.java

```diff
@@ -87,6 +87,12 @@ public boolean equals(Object o)
       return true;
     }
 
+    @Override
+    public int hashCode()
+    {
+      return super.hashCode();
```

Comment:
I think this has the same problem as IdentityExtractionFn. It should return a constant rather than `super.hashCode()`.

Please check the other usages of `super.hashCode()` as well.

Reply:
Oops.. Fixed.